Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ecu_diagnostics"
-version = "0.90.61"
+version = "0.99.0"
```

Comment:
don't skip versions - it gets really confusing. The "rule of thumb": until the project breaks the 1.0.0 barrier (feature complete and stable), increment minor for any breaking change, and any breaking changes are ok.

Reply:
Please use 0.91.0 here